.artifact/tree.json

```diff
@@ -0,0 +1,910 @@
+{
+  "rimage": {
```

Comment:
Do you mind explaining the differences between this and [rdk's .artifact/tree.json](https://github.com/viamrobotics/rdk/blob/fcae3e91d937d54cf84bd1ee80374b09898cb7f5/.artifact/tree.json)?

Reply:
I copied/pasted RDK's tree.json and deleted what I didn't need.